packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -4757,4 +4759,32 @@ void main() {
     expect(selection!.baseOffset, 6);
     expect(selection!.extentOffset, 14);
   }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS, TargetPlatform.android }));
+
+  testWidgets('Ellipsis is passed to the inner TextPainter when provided', (WidgetTester tester) async {
```

Comment:
It's probably easier to add a test to "text_painter_test.dart" to verify the defaulting is working (so you don't have to expose the painter on `RenderEditable`), since there's no real code change in `editable.dart`. 

If you want to make sure eclipsing works then maybe add a golden test?